<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QWidget" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>60</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>141</width>
     <height>521</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>2</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>头像</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>管理员</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_index">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>175</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>首页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_info">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>信息管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_release">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>285</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>发布通知</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>741</width>
     <height>531</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="tab_index">
    <attribute name="title">
     <string>首页</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>当前位置： 首页</string>
     </property>
    </widget>
    <widget class="QChartView" name="chartWidget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>701</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>701</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_info">
    <attribute name="title">
     <string>信息管理</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>681</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>591</width>
       <height>41</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton_search">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>9</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_search">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_add">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>9</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modify">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>9</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_delete">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>9</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_export">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>导出</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_import">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>9</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>导入</string>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>请选择信息</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>体温信息</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>学生信息</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>宿舍信息</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>学院信息</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>宿管信息</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>管理员信息</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>当前位置： 信息管理</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_release">
    <attribute name="title">
     <string>发布通知</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_title">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>请输入标题</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_content">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>281</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_release_confirm">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>380</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>发布</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_release_result">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>390</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>90</y>
     <width>201</width>
     <height>498</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>368</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <attribute name="label">
     <string>公告一</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLineEdit" name="lineEdit_1"/>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_1">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>查看</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>368</height>
     </rect>
    </property>
    <attribute name="label">
     <string>公告二</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_2"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>查看</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>368</height>
     </rect>
    </property>
    <attribute name="label">
     <string>公告三</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_3"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>查看</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>368</height>
     </rect>
    </property>
    <attribute name="label">
     <string>公告四</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_4"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>查看</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>368</height>
     </rect>
    </property>
    <attribute name="label">
     <string>公告五</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_5"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>查看</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>60</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>公告通知</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
